identity-apps-core/apps/authentication-portal/src/main/webapp/login.jsp

```diff
@@ -1052,6 +1059,37 @@
                             <br>
                             <%
                                         }
+                                for (String localAuthenticator : localAuthenticatorNames) {
+                                    if (registeredLocalAuthenticators.contains(localAuthenticator)) {
+                                        continue;
+                                    }
+                            %>
+                                <div class="social-login blurring social-dimmer">
+                                <div class="field">
+                                        <button
+                                            type="button"
+                                            id="icon-<%=iconId%>"
+                                            class="ui button secondary"
+                                            data-testid="login-page-sign-in-with-<%=localAuthenticator%>"
+                                            onclick="handleNoDomain(this,
+                                                    '<%=Encode.forJavaScriptAttribute(Encode.forUriComponent(idpEntry.getKey()))%>',
+                                                    '<%=localAuthenticator%>')"
+                                        >
+                                        <img
+                                            class="ui image"
+                                            src="libs/themes/default/assets/images/authenticators/<%=localAuthenticator%>.svg"
+                                            alt="<%=localAuthenticator%> Logo"
+                                            role="presentation">
+                                        <span>
+                                            <%=AuthenticationEndpointUtil.i18n(resourceBundle, "sign.in.with")%>
+                                            <%=AuthenticationEndpointUtil.i18n(resourceBundle, localAuthenticator)%>
+                                        </span>
+                                        </button>
+                                </div>
+                            </div>
+                            <br>
```

Comment:
```suggestion
                              <div class="social-login blurring social-dimmer">
                                    <div class="field">
                                            <button
                                                type="button"
                                                id="icon-<%=iconId%>"
                                                class="ui button secondary"
                                                data-testid="login-page-sign-in-with-<%=localAuthenticator%>"
                                                onclick="handleNoDomain(this,
                                                        '<%=Encode.forJavaScriptAttribute(Encode.forUriComponent(idpEntry.getKey()))%>',
                                                        '<%=localAuthenticator%>')"
                                            >
                                                <img
                                                    class="ui image"
                                                    src="libs/themes/default/assets/images/authenticators/<%=localAuthenticator%>.svg"
                                                    alt="<%=localAuthenticator%> Logo"
                                                    role="presentation">
                                                <span>
                                                    <%=AuthenticationEndpointUtil.i18n(resourceBundle, "sign.in.with")%>
                                                    <%=AuthenticationEndpointUtil.i18n(resourceBundle, localAuthenticator)%>
                                                </span>
                                            </button>
                                    </div>
                            </div>
                            <br>
```

Reply:
If we can let's format the div elements correctly

Reply:
Fixed by - https://github.com/wso2/identity-apps/pull/6889/commits/8f8f8906694e045debe1f27475a21b04be0c8d9a